angular-workspace/package.json

```diff
@@ -21,24 +21,24 @@
     "format": "ng lint --fix"
   },
   "dependencies": {
```

Comment:
Any thoughts come to mind on how the directive upgrade process should work?

Initial thought:

1. navigate to each file in third party based on tag, i.e.: https://github.com/angular/angular/blob/15.2.0/packages/forms/src/directives/checkbox_value_accessor.ts
2. Take the file contents and do a diff against what is in nimble to merge and changes. Use our [Nimble] comment annotations
3. Update the URL at the top of the file in nimble to know what version of Angular the file was last compared against

Reply:
The process you outlined makes sense to me. It may also be helpful in some cases to diff the two unmodified Angular files against each other to understand what the changes were made upstream that need to be integrated into nimble.

Reply:
Note that we have some docs about the directive upgrade process already [in the README for the `thirdparty` folder](https://github.com/ni/nimble/blob/main/angular-workspace/projects/ni/nimble-angular/src/thirdparty/README.md). At the very least its link needs to be updated but incorporating these instructions would be good too.

One other thought: if Angular ever refactors their tests then it's possible we would no longer need the files that this PR is adding to `utils` or `testing`. It might be good to add a step to either scan the Angular repo for deletions or the Nimble repo for orphaned files.

Reply:
It looks like the docs were already pretty good, so I updated the link and added an additional step about making sure all the files were still required.